airflow/jobs.py

```diff
@@ -603,8 +603,11 @@ def manage_slas(self, dag, session=None):
                             execution_date=dttm,
                             timestamp=ts))
                         Stats.incr('task_sla_miss', 1, 1, tags=['dag_id:{}'.format(ti.dag_id)])
+                        task_id_entity = '.'.join(ti.task_id.split('.')[:-1])
+                        task_id_action = '.'.join(ti.task_id.split('.')[-1:])
                         Stats.incr('task_sla_miss.by_task', 1, 1, tags=['dag_id:{}'.format(ti.dag_id),
-                                                                        'task_id:{}'.format(ti.task_id)])
+                                                                        'task_id:{}'.format(task_id_entity),
```

Comment:
Maybe `task_id` -> `task_entity` or `task_id_prefix` to reduce confusion. I can't think of anything else.